src/validation.cpp

```diff
@@ -2041,6 +2041,16 @@ bool CChainState::FlushStateToDisk(
                last_prune = std::max(1, std::min(last_prune, index.GetSummary().best_block_height));
             });
 
+            for (auto const& blocker : m_blockman.m_prune_blockers) {
+                if (blocker.second) {
```

Comment:
b5a239390fd1b31c7e6107a2e6c35658a2b4697c

How can it be `nullptr`?

Reply:
I guess it should never be `nullptr`. Dropped that if statement.